### ISSUE TITLE
Duration is reported as NaN on testSuiteFinished

### ISSUE THREAD

**Issue by ratherblue** [travisjeffery/mocha-teamcity-reporter#10]
Added this reporter to some of my unit tests and I'm getting output like this:

```
##teamcity[testSuiteFinished name='messageRow(context, options)' duration='NaN']
##teamcity[testSuiteFinished name='Helpers' duration='NaN']
##teamcity[testSuiteFinished name='templateUtils' duration='NaN']
```

**Comment by edendramis** [comment]
I have a fix for it, but wanted to report the issue first.

**Comment by jamie-sherriff** [comment]
Fixed in 3.0.0 release
